src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -193,12 +193,16 @@ export default class HaAutomationActionRow extends LitElement {
           </div>
           ${this._warnings
             ? html`<div class="warning">
-                UI editor is not supported for this config:
+                ${this.hass.localize("ui.errors.config.editor_not_supported")}:
                 <br />
-                <ul>
-                  ${this._warnings.map((warning) => html`<li>${warning}</li>`)}
-                </ul>
-                You can still edit your config in YAML.
+                ${this._warnings!.length > 0 && this._warnings![0] !== undefined
```

Comment:
Is this possible? When does it happen?

Reply:
When you fire the event `"ui-mode-not-available"` without any detailed `Error`. The error handling still creates an entry in `warnings` which is undefined. I tried to suppress that, but somehow failed.

Reply:
Something like?
```
    if (ev.detail && ev.detail instanceof Error) {
      this._warnings = handleStructError(this.hass, ev.detail).warnings;
    }
```

Reply:
You should not fire `ui-mode-not-available` without Error to begin with ofc...

Reply:
Ideally not, but if someone did, then this will handle it. Your code proposal would prevent the error/info from being shown at all, which I think is not ideal.